tika-core/src/main/resources/org/apache/tika/mime/custom-mimetypes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This type should be added to [`tika-mimetypes.xml`](https://github.com/apache/tika/blob/master/tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml) instead of `custom-mimetypes.xml` which is intended for downstream users IIRC. @tballison, could you confirm this please if I am correct.